E2_ESRGAN/lib/model.py

```diff
@@ -46,3 +45,29 @@ def call(self, input_):
     feature = self.lrelu(self.conv_last_1(feature))
     out = self.conv_last_2(feature)
     return out
+
+
+class VGGArch(tf.keras.Model):
+  def __init__(self, output_shape=[1], num_features=64, use_bias=True):
+
+    super(Discriminator, self).__init__()
+    self.conv = lambda n, s, x: tf.keras.layers.Conv2D(
+        n, kernel_size=[3, 3], strides=[s, s], use_bias=use_bias)(x)
+    self.nf = num_features
```

Comment:
`self.num_features`. Names like `nf` get quickly hard to disambiguate once the code gets large enough :-)

Reply:
Fixed it :)